doc/classes/MultiplayerPeerCustom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="MultiplayerPeerCustom" inherits="NetworkedMultiplayerPeer" version="3.5" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A [NetworkedMultiplayerPeer] implementation that can be controlled from a script.
+	</brief_description>
+	<description>
+		A [NetworkedMultiplayerPeer] implementation that can be used as a [member MultiplayerAPI.network_peer] and controlled from a script.
+		Its purpose is to allow adding a new backend for the high-Level multiplayer API without needing to use GDNative.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="deliver_rpc">
+			<return type="void" />
+			<argument index="0" name="buffer" type="PoolByteArray" />
+			<argument index="1" name="from_peer_id" type="int" />
+			<description>
+				Deliver an RPC to the local [MultiplayerAPI].
```

Comment:
should probably mention this is raw bytes, and might not be an RPC (e.g. multiplayer.`send_bytes`).